app/pages/resource/resource-auth-mapping.js

```diff
@@ -0,0 +1,30 @@
+import facebookIcon from '../../assets/icons/facebook.svg';
+import googleIcon from '../../assets/icons/google.svg';
+import mobileIcon from '../../assets/icons/mobile.svg';
+import suomiFiIcon from '../../assets/icons/suomiFi.svg';
+import yleIcon from '../../assets/icons/yle.svg';
+
+/*
+Mapping of resource authentication level with the corresponding allowed login methods.
+If this mapping is updated backend mapping needs updating too and vice version.
+The order of loginMethodNames and loginMethodIcons should match since the title/alt for
+icons is used from loginMethodNames.
+*/
+export const RESOURCE_AUTHENTICATION_GROUPING = {
+  'strong': {
+    loginMethodNames: ['Suomi.fi'],
+    loginMethodIcons: [suomiFiIcon],
+  },
+  'piki': {
+    loginMethodNames: ['PIKI-kirjastokortti'],
+    loginMethodIcons: [],
+  },
+  'mid': {
+    loginMethodNames: ['Phone', 'Suomi.fi'],
+    loginMethodIcons: [mobileIcon, suomiFiIcon],
+  },
+  'weak': {
+    loginMethodNames: ['Suomi.fi', 'Google', 'Facebook', 'Yle Tunnus'],
```

Comment:
Add the stronger levels so they are also available.

Reply:
Added the missing upper levels.